header/department.h

```diff
@@ -9,7 +9,7 @@ using namespace std;
 class Department {
 public:
 	vector<Course> courseList;
-	vector<Instructor> instructiorList;
+    vector<Instructor> instructorList;
```

Comment:
Weird Spacing?

Reply:
Looks normal in both VS and QT, only GitHub does this.